src/Controller/HomepageController.php

```diff
@@ -12,7 +14,7 @@ class HomepageController extends AbstractController
     public function index(): Response
     {
        return $this->render('homepage/index.html.twig', [
-            'controller_name' => 'HomepageController',
+            'controller_name' => $this::class,
```

Comment:
Same here, I would keep it as-is for now, but maybe can you contribute this change to MakerBundle?